README.md

```diff
@@ -1,9 +1,9 @@
-# Example AnyChannel integration with Wordpress using Node Express
+# Example AnyChannel integration with Wordpress using Node Express (Tutorial Version)
 
 ## Description
 This repository contains the source code for an integration service that connects
-Zendesk to Wordpress.  It is intended to be used with the AnyChannel tutorial
-(TODO: Add link once it's available.)
+Zendesk to Wordpress.  It is intended to be used with the AnyChannel tutorial.
+The code referred by the tutorial is commented out in this version. The tutorial has instruction on when to uncomment the commented out code.
```

Comment:
nit: "The code referred by the tutorial is commented out in this version. The tutorial has instruction on when to uncomment the commented out code." => "The tutorial instructs you to implement functionality in the wordpress.js file. Relevant code is already present in wordpress.js, but is commented out." or something like that?